fixtures/elm-home/0.19.0/package/elm/core/1.0.2/src/Char.elm

```diff
@@ -36,17 +47,26 @@ import Elm.Kernel.Char
 {-| A `Char` is a single [unicode][u] character:
 
     'a'
+
     '0'
+
     'Z'
+
     '?'
+
     '"'
+
     'Σ'
+
     '🙈'
 
     '\t'
-    '\"'
+
+    '"'
+
     '\''
-    '\u{1F648}' -- '🙈'
+
+    '🙈' -- '🙈'
```

Comment:
This line was `'\u{1F648}' -- '🙈'`, but elm-format transformed the unicode on the left. I think this line may be discarded, since it was there to add clarity to the docs, right?